cmd/podman/load.go

```diff
@@ -92,7 +93,7 @@ func loadCmd(c *cliconfig.LoadValues) error {
 				return errors.Errorf("error copying file %v", err)
 			}
 
-			input = outFile.Name()
+			c.Input = outFile.Name()
```

Comment:
The reason I set `input` is because of the `parse.ValidateFileName()` below. Will `""` validate? I can't test right now because I'm seeing a major crash in this form of usage and am very low on time right now (will have more time this afternoon).

Reply:
I removed input altogether, now I just if c.Input > 0, then validate, otherwise I generate it from a stdin.